mp/src/game/client/momentum/c_mom_player.h

```diff
@@ -52,6 +52,9 @@ class C_MomentumPlayer : public C_BasePlayer, public CMomRunEntity
     CNetworkVar(int, m_iDirection); // Used in kickback effects for player
     CNetworkVar(int, m_iLastZoomFOV); // Last FOV when zooming
     CNetworkVar(bool, m_bSurfing);
+    CNetworkVar(bool, m_bJumpToggled); // Used by keypress hud
+    CNetworkVar(bool, m_bSprintToggled); // Used by keypress hud
+    CNetworkVar(bool, m_bWalkToggled);   // Used by keypress hud
```

Comment:
I feel like this can be simplified with a `CNetworkVar(int, m_iButtonsToggled)`, and re-using the IN_SPRINT / WALK / etc. Saves the need for 3 separate bools.